Update the automated install guide to cover private macOS install (#422)
swiftlang/swiftly: Documentation/SwiftlyDocs.docc/automated-install.md

### Download the binary

First, download the swiftly binary from swift.org for your operating system (for example, Linux) and processor architecture (for example `arm64` or `x86_64`).
Here's an example using the popular curl command. Note that for GPG signature checking you will first need to install the [swift.org signatures in your toolchain](https://www.swift.org/keys/active/).

```
curl -L https://download.swift.org/swiftly/linux/swiftly-$(uname -m).tar.gz > swiftly.tar.gz
curl -L https://download.swift.org/swiftly/linux/swiftly-$(uname -m).tar.gz.sig > swiftly.tar.gz.sig
gpg --verify swiftly.tar.gz.sig swiftly.tar.gz
tar zxf swiftly.tar.gz
```

On macOS, you can download the pkg file and extract it like this from the command-line:

```
curl -L https://download.swift.org/swiftly/darwin/swiftly.pkg > swiftly.pkg
installer -pkg swiftly.pkg -target CurrentUserHomeDirectory
```

> You could be installing swiftly on a shared user account for macOS. In that case you might want to extract the pkg in a private location where it cannot be affected by other processes running in the same user account.

Reviewer: This would be easier to parse if you laid this out directly:

```suggestion
If you are installing swiftly on a shared user account for macOS, extract the `pkg` in a private location where it cannot be affected by other processes running in the same user account.
```
